app/controllers/api/v1/contacts_controller.rb

```diff
@@ -0,0 +1,31 @@
+module Api
+  module V1
+    class ContactsController < ApplicationController
+      before_action :authenticate_user
+
+      def index
+        contacts = @current_user.contacts
+        if contacts.empty?
+          render json: { data: [], message: "No contacts found" }, status: :ok
+        else
+          render json: ContactsSerializer.new(contacts), status: :ok
+        end
+      end
+
+			def create 
```

Comment:
Styling on this is a little off. Looks extra indented, but otherwise great simple to read actions

Reply:
Agree about the indentation, I think this should be standardized to 2 spaced before merging.

Reply:
We have a 2 spaced set up so not sure why it is off. I will need to check into this.

Reply:
I found the problem, VS Code was set correctly and looked correct on my screen. However Github had my tab/spaces to 8 so when I pushed up, Github made the changes and that is why the code spacing was so exaggerated.